.github/workflows/build.yml

```diff
@@ -2,23 +2,23 @@
 name: Build
 
 on:
-  push:
-    branches-ignore:
-      - "l10n_master"
-      - "gh-pages"
-    paths-ignore:
-      - ".github/workflows/**"
   workflow_dispatch:
+  push:
+    branches:
+      - "main"
+      - "rc"
+      - "hotfix-rc"
+  pull_request:
```

Comment:
❓ The way I read this is that you want to build on any push to main/rc/hotfix-rc, and all pull requests? The main thing I'm concerned about is that with nothing listed under `pull_requests`, that it's not clear PR branches will be built. Assuming I read it correctly, a comment indicating the behavior being relied upon here would be nice.

Reply:
That's correct, that an open-ended trigger applies to all of that type.